source/howtos/edit_your_dockerfile.rst

```diff
@@ -7,6 +7,16 @@ up your container.
 
 A helpful resource is the |bestPractices_link| guide.
 
+The first two lines of the Dockerfile look something like:
+
+:: 
+
+    FROM kbase/kbase:sdkbase2.latest
```

Comment:
Should this actually be `kbase/sdkbase2:python`?

Reply:
After I wrote the above, I read in slack that .latest was for python 2 and .python was for python3